src/components/Icon/Sort.tsx

```diff
@@ -2,9 +2,10 @@ import React from 'react'
 
 type SortProps = {
   rotate?: number
+  color?: string
 }
 
-function Sort({ rotate = 0 }: SortProps) {
```

Comment:
I like this approach but maybe we can have a type matching the colors in our "design system". So it can receive "black-400" or Colors.Black400 as the color prop.